research/gam/README.md

```diff
@@ -17,26 +17,46 @@ likely to have the same label, thus guiding its parameters to a better local
 optima. The classification and agreement models are trained jointly in a
 co-training fashion.
 
-This code repository contains an implementation of Graph Agreement Models. The
-code is organized into the following folders:
+This code repository contains an implementation of Graph Agreement Models [3].
```

Comment:
Can we move this to the beginning of the write-up. The reference number would also have to change to [1].

Reply:
Done.